src/view/com/notifications/Feed.tsx

```diff
@@ -112,6 +123,13 @@ export function Feed({
     fetchNextPage()
   }, [fetchNextPage])
 
+  const onSelectTab = React.useCallback(
+    (index: number) => setTab(tabs[index]),
+    [setTab],
+  )
+
+  const tabLabels = React.useMemo(() => tabs.map(({label}) => label), [])
```

Comment:
💅  This can be moved out of the component (under `const tabs`) since it has no deps. _Or_, it could be generated at runtime below:

```js
<TabBar
  items={...}
```

Performance nit for an Array of size 2!